docs/DEVELOPMENT.md

```diff
@@ -0,0 +1,402 @@
+# Development
+
+Table of Content
+<!-- vscode-markdown-toc -->
+* 1. [Project Requirements](#ProjectRequirements)
+* 2. [Code Submission Process](#CodeSubmissionProcess)
+* 3. [Local Environment Setup](#LocalEnvironmentSetup)
+* 4. [Develop and Test on a GKE cluster](#DevelopandTestonaGKEcluster)
+* 5. [Advanced Skaffold commands](#AdvancedSkaffoldcommands)
+* 6. [Debugging](#Debugging)
+* 7. [Testing](#Testing)
```

Comment:
The numbering for the TOC looks different when viewed in the browser

Reply:
Updated: https://github.com/GoogleCloudPlatform/core-solution-services/blob/e1f369224569cf1d6966616c2f76a78f75a8cd97/docs/DEVELOPMENT.md